README.md

```diff
@@ -1,3 +1,19 @@
-### Has an animated splash screen
+# README
+
+## Latest updates:
+### Main Screen Prototype
+
+This link should take you to a Mockflow prototype I made. Image is below.
+I'm sure others have been working on their ideas so this is just an example of
+something we could do!
+
+Link: https://wireframepro.mockflow.com/view/Mdf63e816b66eb76a30b58238a3a3abd91556497828194
+
+![LauraMockUp](https://user-images.githubusercontent.com/47276506/56872875-1ff3de00-6a04-11e9-8209-02fb31a26af9.png)
+
+### Splash screen:
+
+The gif has been made but it does not work on Android and needs better resolution
+=======
+To run, install expo CLI on your computer and the expo client app on your phone. See https://expo.io/learn
```

Comment:
Link to here for the installation procedure:

https://github.com/reisman-internship-2019-greenmap/GreenMap-app/wiki/App-Installation-Process

And put the installation on the top along with standard info about what the app repo is for. The new updates can then be posted below that, or can also go into the wiki (I think readme is fine for now and we can move older updates into the wiki to track development over time.